### PR TITLE
Editorialize main schema intro

### PR DESCRIPTION
**Pull request by LPardue**

### CODE REVIEW

File: draft-ietf-quic-qlog-main-schema.md [hunk 1]
```diff
@@ -48,53 +48,63 @@ informative:
...
-
-This document aims to provide a high-level schema and harness that describes the
-general layout of an easily usable, shareable, aggregatable and structured logging
-format. This high-level schema is protocol agnostic, with logging entries for
-specific protocols and use cases being defined in other documents (see for example
-{{QLOG-QUIC}} for QUIC and {{QLOG-H3}} for HTTP/3 and QPACK-related event
-definitions).
+Endpoint logging is a useful strategy for capturing and understanding how
+applications using network protocols are behaving, particularly where protocols
+have an encrypted wire image that restricts observers ability to see what is
```
Comment (rmarx) [inline draft-ietf-quic-qlog-main-schema.md:77]:
```suggestion
have an encrypted wire image that restricts observers' ability to see what is
```

File: draft-ietf-quic-qlog-main-schema.md [hunk 2]
```diff
@@ -48,53 +48,63 @@ informative:
...
+happening.
+
+Many applications implement logging but the format tends to be custom rather than
+standardized. This has a effect on the tools and methods that are used to
```
Comment (rmarx) [inline draft-ietf-quic-qlog-main-schema.md:81]:
```suggestion
standardized. This has an effect on the tools and methods that are used to
```

File: draft-ietf-quic-qlog-main-schema.md [hunk 3]
```diff
@@ -48,53 +48,63 @@ informative:
...
+analyze the logs, for example to perform root cause analysis of an
+interoperability failure between distinct implementations. A lack of a common
+format impedes the development of common tooling that can be used by all parties
+that have access to logs.
+
+This document defines qlog, an extensible high-level schema and harness that
+provides a shareable, aggregatable and structured logging format intended to be
+more usable. This high-level schema is independent of protocol, with logging
```
Comment (rmarx) [inline draft-ietf-quic-qlog-main-schema.md:89]:
```suggestion
provides a shareable, aggregatable and structured logging format. This high-level schema is independent of protocol, with logging
```

"more usable" is vague... more usable than what? I propose just cutting this

File: draft-ietf-quic-qlog-main-schema.md [hunk 4]
```diff
@@ -1156,10 +1166,10 @@ SimulationMarker = {
...
 
 This document and other related qlog schema definitions are intentionally
-serialization-format agnostic. This means that implementers themselves can choose
-how to represent and serialize qlog data practically on disk or on the wire. Some
-examples of possible formats are JSON, CBOR, CSV, protocol buffers, flatbuffers,
-etc.
+independent of serialization format. This means that implementers themselves can
+choose how to represent and serialize qlog data practically on disk or on the
+wire. Some examples of possible formats are JSON, CBOR, CSV, protocol buffers,
+flatbuffers, etc.
```
Comment (rmarx) [inline draft-ietf-quic-qlog-main-schema.md:1172]:
```suggestion
wire. Some examples of possible formats are JSON, CBOR, CSV, protobuf,
flatbuffers, etc.
```

Consistency with abstract

**Review by rmarx — CHANGES_REQUESTED** [review]

File: draft-ietf-quic-qlog-main-schema.md (hunk 3 above)
Comment (LPardue) [reply draft-ietf-quic-qlog-main-schema.md:89]: me too!

File: draft-ietf-quic-qlog-main-schema.md [hunk 5]
```diff
@@ -48,53 +48,62 @@ informative:
...
-format. This high-level schema is protocol agnostic, with logging entries for
-specific protocols and use cases being defined in other documents (see for example
-{{QLOG-QUIC}} for QUIC and {{QLOG-H3}} for HTTP/3 and QPACK-related event
-definitions).
+Many applications implement logging but the format tends to be custom rather than
+standardized. This has an effect on the tools and methods that are used to
+analyze the logs, for example to perform root cause analysis of an
+interoperability failure between distinct implementations. A lack of a common
+format impedes the development of common tooling that can be used by all parties
+that have access to logs.
```
Comment (marten-seemann) [inline draft-ietf-quic-qlog-main-schema.md:85]:
```suggestion
Many applications implement logging using a custom logging format.
This has an effect on the tools and methods that are used to
analyze the logs, for example to perform root cause analysis of an
interoperability failure between distinct implementations. A lack of a common
format impedes the development of common tooling that can be used by all parties
that have access to logs.
```

**Review by marten-seemann — APPROVED** [review]

**Review by rmarx — APPROVED** [review]

**Pull request merged**